### ISSUE TITLE
FR: Pass in multiple users

### ISSUE THREAD

**Issue by hemanth** [SamVerschueren/dev-time-cli#2]
It will be useful if this util takes a list of devs.

**Comment by sindresorhus** [comment]
Just do bash loop?

**Comment by hemanth** [comment]
Well, `echo passy sindresorhus | xargs -n1 dev-time` vs `dev-time --user=passy,sindersorhus` and we could also have the output in the form of `<user>:<time>` maybe?

**Comment by sindresorhus** [comment]
I guess it could be useful if you want to have a meeting or something and want to see people's time.

> dev-time --user=passy,sindersorhus and we could also have the output in the form of <user>:<time> maybe?

:+1:

**Comment by SamVerschueren** [comment]
Which one do you prefer?

```
$ dev-time sindresorhus
12:37 - 10 Dec. 2015

$ dev-time sindresorhus hemanth
sindresorhus
     12:37 - 10 Dec. 2015
hemanth
     17:07 - 10 Dec. 2015
```

Or for multiple users with a flag (`--users,-u`)?

```
$ dev-time -u=sindresorhus,hemanth
sindresorhus
     12:37 - 10 Dec. 2015
hemanth
     17:07 - 10 Dec. 2015
```

**Comment by hemanth** [comment]
`$dev-time sindresorhus hemanth` is more intuitive for me.

Output could be like:

```
sindresorhus: 12:37 - 10 Dec. 2015
hemanth: 17:07 - 10 Dec. 2015
```

**Comment by SamVerschueren** [comment]
Don't like the output that much. Very unreadable when having a lot of names with different length.

**Comment by hemanth** [comment]
Agree, I was thinking in terms of parseable format.

**Comment by SamVerschueren** [comment]
If someone comes up with a use case to have it parseable, we might think about a `--json` flag.

**Comment by hemanth** [comment]
Ok usecase: Lets say we have a webpage that lists all the members of an github org, will their current time ;)  [Wired usecase, this is the first thing that came to my mind]

**Comment by SamVerschueren** [comment]
@hemanth create a new issue for that one ;).

**Comment by hemanth** [comment]
Done #7 ;)

**Comment by SamVerschueren** [comment]
What do you think?

![dev-time](https://cloud.githubusercontent.com/assets/1913805/11715709/296f3bb6-9f44-11e5-95a9-94710fed83db.gif)

**Comment by hemanth** [comment]
Sweet! :+1:

**Comment by hemanth** [comment]
@SamVerschueren Are you bumping the version now?

**Comment by SamVerschueren** [comment]
Fixing the tests first :).

**Comment by hemanth** [comment]
Cool, just noticed.

**Comment by SamVerschueren** [comment]
Released

**Comment by hemanth** [comment]
```
$ dev-time --version
1.2.0
```

Isn't notifying me to update to the latest version, you were using `update-notifier` right?

**Comment by SamVerschueren** [comment]
Yes I'm using it here [cli.js#L47](https://github.com/SamVerschueren/dev-time-cli/blob/master/cli.js#L47). Not sure why it isn't notifying. Probably it is checking it only once a day (the first time it is used) as it uses `configstore` to store the last checked time.

https://github.com/yeoman/update-notifier#updatecheckinterval
